site/docs/about/roadmap.mdx

```diff
@@ -94,3 +43,23 @@ import LargeParagraph from "../../src/components/LargeParagraph";
 
 
 - **HDS 1.0 Release**
+
+**Q2 2021**
+- Image usage instructions
+- Tone of voice instructions
+- Basic tables
+- Content higlights, such as hero and liftup cards
+- Breadcrumbs
+- Sliders
+
+- Pattern: Multi-step forms
+- Pattern: Cookie Consent 
+
+**Q3 2021**
+- Map components
+- Opening hours components
+
+**Q4 2021**
```

Comment:
Same with this - it has a different list spacing than other sections.

![image](https://user-images.githubusercontent.com/4214671/109621598-db778300-7b43-11eb-9321-502164f0cccd.png)